openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1507,7 +1507,96 @@ It is up to the Credential Issuer whether to update both the signature and the c
 
 # Privacy Considerations
 
-TBD
+When [@!RFC9396] is used, the Privacy Considerations of that specification also apply.
+
+The privacy principles of [@ISO.29100] should be adhered to.
+
+## User Consent
+
+The Credential Issuer should obtain the End-User's consent before issuing Credential(s)
+to the Wallet. It should be made clear to the End-User what information is being included in the
+Credential(s) and for what purpose.
```

Comment:
Why lower-case should here?

Reply:
It used to be a `MUST`, we had a passionate discussion during the WG call based on [a comment here](https://github.com/openid/OpenID4VCI/pull/244#discussion_r1473908607) and there were concerns with having a normative statements regarding consent.

Reply:
should all `should`s in this text be upper case `SHOULD`?